.bazelrc

```diff
@@ -26,7 +22,7 @@ build --cxxopt=-std=c++14
 build --sandbox_tmpfs_path=/tmp
 
 # Strict dependency check for C++ includes.
-build --features=layering_check
```

Comment:
Why no longer make this part of the default build expectations?

Reply:
Since I see that `--config=ci` is not used everywhere in bazel_test.yml, let's document here the use of `--config=ci`.